### ISSUE TITLE
Parent Event Not Firing

### ISSUE THREAD

**Issue by MafcoCinco** [domino-clj/domino#2]
Given this test schema:

```clojure
(def test-schema {:model [[:foo {:id :foo}
                           [:bar {:id :bar}]]
                          [:baz {:id :baz}]]

                  :events [{:inputs [:baz]
                            :outputs [:bar]
                            :handler (fn [ctx [baz] _]
                                       [(inc baz)])}]

                  :effects [{:inputs [:foo]
                             :handler (fn [ctx [foo]]
                                        (println (str "Received `foo` effect" \newline
                                                      "---------------------" \newline
                                                      (pr-str foo))))}]})
```
I would expect the effect handler for `:foo` to fire when I executed the following code:

```clojure
clojure.user> (def ctx (atom (domino/initialize! test-schema {})))
;; ctx
clojure.user> (domino/transact @ctx [[[:baz] 1]])
```
The new `:domino.core/db` is as follows:
```clojure
{:baz 1
 :foo {:bar 2}}
```
However, the `println` in the `:effects` handler for `:foo` is not executed.  Is this a misunderstanding on my part WRT how events are supposed to cascade or is this a bug?

Thanks!

**Comment by MafcoCinco** [comment]
Looking more at the context, the change history is as follows:

```clojure
{:change-history [[[:baz] 1] [[:foo :bar] 2]]}
```
I would argue that `[[:foo] {:bar 2}]` should be part of this change history.

**Comment by MafcoCinco** [comment]
This has been fixed as of version `0.2.0`.
